docs/dapp/emerald/README.mdx

```diff
@@ -1,3 +1,7 @@
+---
+description: ABC
```

Comment:
TODO?

Reply:
Oops, just some debugging leftover (is category description like emerald shown when used as doccard => no).